src/plugins/mer/mersettings.cpp

```diff
@@ -115,6 +115,74 @@ void MerSettings::setRpmValidationByDefault(bool byDefault)
     emit s_instance->rpmValidationByDefaultChanged(s_instance->m_rpmValidationByDefault);
 }
 
+QHash<QString, MerEmulatorDeviceModel> MerSettings::customEmulatorDeviceModels()
+{
+    Q_ASSERT(s_instance);
+
+    return s_instance->m_customEmulatorDeviceModels;
+}
+
+bool MerSettings::isCustomEmulatorDeviceModel(const QString &name)
+{
+    Q_ASSERT(s_instance);
+
+    return s_instance->m_customEmulatorDeviceModels.contains(name);
+}
+
+MerEmulatorDeviceModel MerSettings::customEmulatorDeviceModel(const QString &name)
+{
+    Q_ASSERT(s_instance);
+
+    return s_instance->m_customEmulatorDeviceModels.value(name);
+}
```

Comment:
I would prefer using MerSettings::customEmulatorDeviceModels().contains() and .value() instead of introducing this indirection.

Reply:
This code removed.